pages/common/aws-help.md

```diff
@@ -0,0 +1,16 @@
+# aws help
+
+> Display help information about AWS CLI.
+> More information: <https://docs.aws.amazon.com/cli/latest/reference/index.html>.
+
+- Display help:
+
+`aws help`
+
+- Display a list of all available topics:
+
+`aws help topics`
+
+- Display help about a specific topic:
+
+`aws help {{topicname}}`
```

Comment:
```suggestion
`aws help {{topic_name}}`
```

Follow snake_case as mentioned in [style guide](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#token-syntax).